database/seeds/SourceServiceTableSeeder.php

```diff
@@ -319,6 +364,33 @@ private function addArkVariables()
         ]);
     }
 
+    private function addCSGOVariables()
+    {
+        ServiceVariable::updateOrCreate([
+            'option_id' => $this->option['csgo']->id,
+            'env_variable' => 'SRCDS_MAP',
+        ], [
+            'name' => 'Map',
+            'description' => 'The default map for the server.',
+            'default_value' => '',
```

Comment:
Is there a map that is always there in the game that can be defaulted to to avoid confusion on this field?